src/qml/filters/color/ui.qml

```diff
@@ -28,19 +28,16 @@ Item {
     width: 450
     height: 250
```

Comment:
The height needs to increase to make canvas larger for proper scrolling when the filter dock is small and shows scroll bars.

Reply:
Fixed .